nixos/modules/services/monitoring/rustdesk-server.nix

```
@@ -0,0 +1,96 @@
+{ lib, pkgs, config, ... }:
+{
+  options.services.rustdesk-server = with lib; with types; {
+    enable = mkEnableOption "enable RustDesk";
+
+    package = mkOption {
+      type = package;
+      description = "The directus package to use";
+      default = pkgs.rustdesk-server;
+    };
```

Just a question - was the "directus" part intentional, or just a copy paste error?

```suggestion
    package = mkPackageOption pkgs "rustdesk-server" { };
```